aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -276,8 +534,9 @@ module aptos_framework::coin {
         coin_store.frozen
     }
 
+    #[deprecated]
     #[view]
-    /// Returns `true` if `account_addr` is registered to receive `CoinType`.
+    /// Always return `true` since `account_addr` can receive `CoinType` in fungible asset without registration.
```

Comment:
should this return true if primary exists but coinstore doesn't?

Reply:
If we consider both, primary FS will always exist (if not, it will be created), so this should return `true` always.
Let me refactor all the PR and AIP to lay out a comprehensive solution.